pkg/genericapiserver/options/serving.go

```diff
@@ -90,7 +90,7 @@ func (s *SecureServingOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.IPVar(&s.ServingOptions.BindAddress, "bind-address", s.ServingOptions.BindAddress, ""+
 		"The IP address on which to listen for the --secure-port port. The "+
 		"associated interface(s) must be reachable by the rest of the cluster, and by CLI/web "+
-		"clients. If blank, all interfaces will be used (0.0.0.0).")
```

Comment:
@deads2k I'm not following. Will `--bind-address ""` result in a different behavior than no `--bind-address` at all?